src/bin/cargo/commands/help.rs

```diff
@@ -34,14 +40,20 @@ fn try_help(config: &Config, subcommand: &str) -> CargoResult<bool> {
     let subcommand = match check_alias(config, subcommand) {
         // If this alias is more than a simple subcommand pass-through, show the alias.
         Some(argv) if argv.len() > 1 => {
-            let alias = argv.join(" ");
+            let alias = argv
+                .into_iter()
+                .map(|s| s.to_string_lossy().into_owned())
```

Comment:
Should this be "aliases are always UTF-8" as well?

Reply:
Looking again at this, I think I can shift where `OsString` handling is done to make this code cleaner.